### ISSUE TITLE
Can you provide the hyper-parameter to train on other datasets?

### ISSUE THREAD

**Issue by scoobym** [lijin118/LisGAN#1]
Really appreciate your work, however, I use `python lisgan.py --dataset AWA1 --nz 85 --attSize 85 --syn_num 300 --nepoch 350` to train on the AWA dataset and the results fluctuated between 0.5 and 0.6(zsl setting), maybe I'm using the wrong hyper-parameter. So could you kindly provide the hyper-parameter to train on other datasets, i.e., AWA, CUB, SUN

**Comment by lijin118** [comment]
Please run the codes with pytorch 0.3.1 and you will get what you are expecting.
You can refer to the updated readme.

**Comment by scoobym** [comment]
I run with the parameter you provide in readme, don't know why only got  unseen class accuracy=  tensor(0.6728) in (22/30)epoch

**Comment by lijin118** [comment]
You mean 0.6728 or 0.6728%?  If it is the former, I think it is normal. Otherwise, please check that you run the codes with pytorch 0.3.1.

**Comment by scoobym** [comment]
Thank you! Finally get the result !

**Comment by naumanbashir70** [comment]
@scoobym 
I appreciate that you  have run the code successfully. Could you help me in executing the code.
It is giving me error  'train_loc' as this is not available in the provided dataset. How you manage to run this or you have you used other dataset?

**Comment by scoobym** [comment]
@naumanbashir70 The problem is that the dataset provided on the current website has changed and I used the original version of the dataset.

**Comment by naumanbashir70** [comment]
@scoobym  thanks for replying.
Could you tell me where I can find original dataset? or could you send me if you have.
Thanks

**Comment by scoobym** [comment]
@naumanbashir70 Sorry, I don't have the original dataset. You can ask author for it.

**Comment by naumanbashir70** [comment]
okay, Thanks
